sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -254,6 +291,14 @@ static TestMode initializeTestMode() {
         return TestingHelpers.getTestMode();
     }
 
+    /**
+     * Indicates whether the out of process test recording proxy is in use.
+     * @return true if test proxy is to be used.
+     */
+    static boolean useTestProxy() {
```

Comment:
can be private?

Reply:
needs to be package private so `TestProxyTestBase` can see it.